src/types/Claim.ts

```diff
@@ -8,7 +8,8 @@ import ICType from './CType'
 import IPublicIdentity from './PublicIdentity'
 
 export default interface IClaim {
-  cType: ICType['hash']
+  cTypeHash: ICType['hash'] // Subject to change TODO: find out alternative to storing whole ICType is necessary
+  cTypeSchema?: ICType['schema']
```

Comment:
The ctypeschema is only used in the constructor of `Claim`. It is not included as a property

Reply:
building from exisitng Claims needs to have the schema to verify the claim against the schema, will be done in the from...() function in future Iterations.